Build fails when libelf is not installed in default location

I have my libelf installation in a non-standard place.  The build fails from the following cmake:
```
cmake -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ \
-DCMAKE_INSTALL_PREFIX=/myinstalldir \
-DCMAKE_BUILD_TYPE=Debug \
-DLIBOMP_OMPD_SUPPORT=TRUE \
-DLIBOMP_OMPT_SUPPORT=TRUE \
-DLIBOMP_OMP_VERSION=45 \
-DLIBOMPTARGET_DEP_LIBELF_INCLUDE_DIR=/libelfinstalldir/include\
-DLIBOMPTARGET_DEP_LIBELF_LIBRARIES=/libelfinstalldir/lib/libelf.a\
-DLIBOMPTARGET_DEP_LIBFFI_LIBRARIES=/libffiinstalldir/libffi.a\
-DLIBOMPTARGET_DEP_LIBFFI_INCLUDE_DIR=/libffiinstalldir/include\
../llvm
```

The build fails as follows:
```
[  1%] Building CXX object projects/openmp/libomptarget/plugins/cuda/CMakeFiles/omptarget.rtl.cuda.dir/src/rtl.cpp.o
[  1%] Building CXX object projects/openmp/libomptarget/plugins/ppc64le/CMakeFiles/omptarget.rtl.ppc64.dir/__/generic-elf-64bit/src/rtl.cpp.o
[  1%] Linking C executable ../../bin/count
In file included from ../llvm/projects/openmp/libomptarget/plugins/cuda/src/rtl.cpp:32:0:
../llvm/projects/openmp/libomptarget/plugins/cuda/src/../../common/elf_common.c:23:20: fatal error: libelf.h: No such file or directory
 #include <libelf.h>
                    ^
compilation terminated.
```

--- a/libomptarget/plugins/cuda/CMakeLists.txt
+++ b/libomptarget/plugins/cuda/CMakeLists.txt
@@ -24,6 +24,7 @@ if(LIBOMPTARGET_DEP_LIBELF_FOUND)
       endif()
       
       include_directories(${LIBOMPTARGET_DEP_CUDA_INCLUDE_DIRS})
+      include_directories(${LIBOMPTARGET_DEP_LIBELF_INCLUDE_DIRS})
     
       add_library(omptarget.rtl.cuda SHARED src/rtl.cpp)
       

